src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -3370,6 +3372,35 @@ public object ConvertBack(object value, Type targetType, object parameter, Syste
         }
     }
 
+    public class Base64ToImageConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            if (value == null) return null;
+
+            string s = string.Empty;
+            if (value is string)
+            {
+                s = value as string;
+            }
+
+            if (string.IsNullOrEmpty(s)) return null;
+
+            BitmapImage bi = new BitmapImage();
+
+            bi.BeginInit();
+            bi.StreamSource = new MemoryStream(System.Convert.FromBase64String(s));
```

Comment:
@zeusongit I think you need to carefully examine if this code leaks. You never dispose this stream.